R/RunStep.R

```diff
@@ -42,18 +42,29 @@ RunStep <- function(lStep, lData, lMeta) {
 
   cli::cli_h3("Evaluating {length(params)} parameter(s) for {.fn {lStep$name}}")
 
-  # This loop iterates over each parameter in the 'params' object.
+  # Allow setting parameters from a list in lData when the 'context' parameter is present in the 'lStep' object.
+  if(!is.null(lStep$context)) {
```

Comment:
@jwildfire Do we have an example of how this `context` field would work? it's in both `lStep` and `lData`, but it seems that the bulk of the information would be in the `lData` element. I'm just not sure i'm completely following how `context` would be used-- seeing an example may be helpful.